test/k8sT/Policies.go

```diff
@@ -595,6 +598,81 @@ var _ = Describe("K8sValidatedPolicyTest", func() {
 				res.ExpectFail("Egress DNS connectivity should be denied for pod %q", pod)
 			}
 		})
+
+		It("Allows traffic with k8s default-allow ingress policy", func() {
+			By("Installing ingress default-allow")
+			_, err := kubectl.CiliumPolicyAction(
+				helpers.KubeSystemNamespace, knpAllowIngress, helpers.KubectlApply, helpers.HelperTimeout)
+			Expect(err).Should(BeNil(),
+				"L3 allow-ingress Policy cannot be applied in %q namespace", helpers.DefaultNamespace)
+
+			By("Checking that all endpoints have ingress enforcement enabled")
+			endpoints, err := kubectl.CiliumEndpointsListByLabel(ciliumPod, podFilter)
```

Comment:
Quick one, I think that we should use CEP commands here.

Reply:
How come? Can we file this as a a GH TODO for our K8s tests?